frontend/src/lib/utils.tsx

```diff
@@ -810,7 +810,7 @@ export const dateMapping: DateMappingOption[] = [
     },
     {
         key: 'Yesterday',
-        values: ['-1dStart', 'dStart'],
```

Comment:
I believe this fix display for the existing "Yesterday" insights though. The fix is preserving the `dStart` option with `inactive: true` and adding this `-1dEnd` as a new item (without `inactive`).

Reply:
Or actually Marius [made a good point](https://posthog.slack.com/archives/C0368RPHLQH/p1685543952850179?thread_ts=1685543082.523449&cid=C0368RPHLQH) that this will shop up as "Custom", which is probably better than showing "Yesterday", given this really is not quite yesterday. So let's leave "Custom".